resources/ts/Pages/Error.tsx

```diff
@@ -3,37 +3,57 @@ import React from "react";
 import Layout from "../layouts/Layout";
 import { Card } from "../Components/Card";
 import { Trans, useTranslation } from "react-i18next";
-import { ButtonSecondary } from "../Components/ButtonSecondary";
-import { Link } from "../Components/Link";
+import { ButtonSecondary } from "../../ts/Components/ButtonSecondary";
+import { Link } from "../../ts/Components/Link";
+import { ErrorImage } from "../Components/ErrorImage";
+import { Heading } from "../Components/Heading";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
 
-const Error = () => {
+type Props = {
+  statusCode: string;
+};
+
+const Error = ({ statusCode }: Props) => {
   const { t } = useTranslation();
   const navigateBack = () => window.history.back();
 
+  if (statusCode === "500")
```

Comment:
```suggestion
  if (statusCode === 500)
```

Reply:
Maybe it is more logical to check for 404 and return the jsx. And the default return at the end is a 500 return